labgrid/util/helper.py

```diff
@@ -106,9 +106,17 @@ def enable_logging(self):
         Loglevel is logging.INFO."""
         self.register(ProcessWrapper.log_callback)
 
+    def disable_logging(self):
+        """Disables process output logging."""
+        self.unregister(ProcessWrapper.log_callback)
```

Comment:
We may need an assert here to detect unbalanced enable/disable?

Reply:
The register() and unregister() methods already have these asserts.